model_compression_toolkit/data_generation/pytorch/model_info_exctractors.py

```diff
@@ -126,6 +131,27 @@ def __init__(self,
                 hook_handle = module.register_forward_hook(hook.hook)# Register the hook on the module
                 self.hook_handles.append(hook_handle)# Store the hook handle in the hook_handles list
 
+        # This code finds the output layers of the model and adds hooks to the input activation of
+        # those layers. We use torch.fx.symbolic_trace to get a static graph to find the output layers.
+        output_nodes = []
+        self.out_module_weights = []
+
+        # Find the output nodes in the graph
+        for node in fx_model.graph.nodes:
```

Comment:
This code can be replaced with a one-liner to create the list, consider it if you like